src/Whoops/Util/TemplateHelper.php

```diff
@@ -183,7 +183,7 @@ public function dump($value)
             return $output;
         }
 
-        return print_r($value, true);
+        return var_dump($value);
```

Comment:
In case you want return the content from var-dump you need a output buffer to capture it

Reply:
I'm not sure where my mind was. `var_dump` should be `var_export`. Even though it works in Whoops, the signature of the method would not make sense.